recipe/meta.yaml

```
@@ -7,32 +7,33 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lipyphilic-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/lipyphilic-{{ version }}.tar.gz
   sha256: db58466f4a78962cd274f19c08dc7413f360ca22ce86f3bea908716a3943657d
 
 build:
-  number: 0
-  noarch: python
+  number: 1
   script:
     - {{ PYTHON }} -m pip install . -vv
     - cd rust
     - cargo-bundle-licenses --format yaml --output ../THIRDPARTY.yml
+  skip: true  # [ py==39 ]
 
 requirements:
   build:
-    - python                                 # [build_platform != target_platform]
+    - python >=3.10,<3.12                    # [build_platform != target_platform]
```

```suggestion
    - python >=3.10,<3.13                    # [build_platform != target_platform]
```
Is there an issue with building with 3.12?

---

There is not! 
It is actually building for 3.12 and it even passed the [tests](https://github.com/conda-forge/lipyphilic-feedstock/pull/14/checks?check_run_id=33900768761) for every OS.
It took me some time to realize the build variants are the ones created at .ci_support with `conda smithy rerender` and this add variants from a few places:
```
Adding in variants from internal_defaults
Adding in variants from /home/rchaves/.cache/conda-smithy/conda_build_config.yaml
Adding in variants from /home/rchaves/repo/lipyphilic-feedstock/recipe/conda_build_config.yaml
Adding in variants from argument_variants
```
I am not sure why it worked with the `<3.12`  in the build requirements, but it is nice having it at least for clarity.